### ISSUE TITLE
Add a link/text telling where to get a token

### ISSUE THREAD

**Issue by xlqian** [CanalTP/navitia-playground#177]
A user of navitia playground said that he was a little lost when he began because he had no idea where to get a token. He finally contacted us and figured out where to get it.

We may add a link(a hint? a tooltip? ) somewhere to tell where to get the token. In fact, we do tell where to get the token on the main page of github, but the user may ignore that carelessly.

Another proposition would be that navitia playground detect a sandbox token and the user is alerted if he's using it... (technically it's bad, because we have to hardcode the sandbox token in our codes...)

**Comment by pbougue** [comment]
A link to the doc part may be another idea? But when to output it...

**Comment by TeXitoi** [comment]
@pbougue there is already a link to the doc at the top of the page. Or I didn't understand what you mean.

**Comment by razum2um** [comment]
@TeXitoi could you also clarify where to get the token if user wants to query selfhosted navitia from docker container on localhost:9191?

**Comment by TeXitoi** [comment]
@razum2um just leave the token field empty.

**Comment by razum2um** [comment]
@TeXitoi yes thanks :) I figured that out and changed the question - can I rollout navitia with token-based auth from docker like on http://www.navitia.io/ ?

**Comment by kinnou02** [comment]
it's easy if you use the compose version: https://github.com/CanalTP/navitia-docker-compose
you just have to set an environment variable for the jormungandr container: https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py#L22
so in docker-compose.yml:
```
  - JORMUNGANDR_IS_PUBLIC=false
```

You will need to create users with tyr: https://github.com/CanalTP/navitia/tree/dev/source/tyr
